README.md

```diff
@@ -1,70 +1,50 @@
-# Getting Started with Create React App
+# City-Weather App
+A react-redux web app that help users to search for weather condition of any city, view location & date, view wind status & wind direction, view humidity, air pressure, view weather condition of the next 5 days, and convert from degree Celsius to Fahrenheit or vice visa
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+### Live Demo
+[City-Weather App:](https://joyful-arithmetic-e662f9.netlify.app/)
 
-## Available Scripts
+[Video Demo:](https://www.loom.com/share/0046e2bc7a3b4553b649f7dcd167f784) 
```

Comment:
Good job on making a video presentation for your project as required by the project and the previous reviewer, but you still have a few more areas to mention in the video that you left out and it is required you mention them in the video.

- [ ] Kindly mention a code section that you are proud of implementing. The idea here is for you to show a basic code section in the project that you feel really proud working on. Please redo the video, and mention that part 👍